src/data/padics/padic_norm.lean

```diff
@@ -491,7 +491,35 @@ The p-adic norm of 1 is 1.
 @[simp] protected lemma one : padic_norm p 1 = 1 := by simp [padic_norm]
 
 /--
-The image of `padic_norm p` is {0} ∪ {p^(-n) | n ∈ ℤ}.
+The p-adic norm of `p` is `1/p` if `p > 1`.
+-/
+lemma padic_norm_p {p : ℕ} (hp : 1 < p) : padic_norm p p = 1 / p :=
+by simp [padic_norm, (show p ≠ 0, by linarith), padic_val_rat.padic_val_rat_self hp]
+
+/--
+An alternate version of `padic_norm.padic_norm_p` that assumes `p` is prime.
+-/
+@[simp] lemma padic_norm_p_of_prime (p : ℕ) [fact p.prime] : padic_norm p p = 1 / p :=
+padic_norm_p $ nat.prime.one_lt ‹_›
+
+/--
+The p-adic norm of `p` is less than 1 if `p > 1`.
+-/
+lemma padic_norm_p_lt_one {p : ℕ} (hp : 1 < p) : padic_norm p p < 1 :=
+begin
+  rw [padic_norm_p hp, div_lt_iff, one_mul],
+  { exact_mod_cast hp },
+  { exact_mod_cast zero_lt_one.trans hp },
+end
+
+/--
+An alternate version of `padic_norm.padic_norm_p_lt_one` that assumes `p` is prime.
+-/
+lemma padic_norm_p_lt_one_of_prime (p : ℕ) [fact p.prime] : padic_norm p p < 1 :=
+padic_norm_p_lt_one $ nat.prime.one_lt ‹_›
+
+/--
+The image of `padic_norm p` is `{0} ∪ {p^(-n) | n ∈ ℤ}`.
 -/
 protected theorem image {q : ℚ} (hq : q ≠ 0) : ∃ n : ℤ, padic_norm p q = p ^ (-n) :=
```

Comment:
This name and docstring suggest the statement will use `set.image`

Reply:
This wasn't changed here, it just looks like it was in the diff because I added something above. I'll try to improve anyway...

Reply:
Renamed to `values_discrete`, and changed `n` to `z`.